ts/create-kpt-functions/tsconfig.json

```diff
@@ -1,21 +1,27 @@
 {
   "compilerOptions": {
-    "module": "commonjs",
-    "target": "es6",
-    "resolveJsonModule": true,
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "baseUrl": ".",
```

Comment:
remove.

Might want to do 'diff' on all  tsconfig.json to make sure they're consistent